src/ds/logger.h

```diff
@@ -27,7 +27,7 @@ namespace logger
   enum Level
   {
     TRACE = 0,
-    DBG, // events useful for debugging
+    DEBUG, // events useful for debugging
```

Comment:
Slightly unrelated, but consistency should help people find the macros (`INFO` => (`LOG_INFO_FMT`, `--log-level="info"`, `[info ]`), `DBG` => ???)